consumer/migration/hermes_migrator.go

```diff
@@ -111,6 +120,16 @@ func (m *HermesMigrator) Start(id string) error {
 	if err != nil {
 		return fmt.Errorf("could not get registry address: %w", err)
 	}
+
+	opened, err := m.isChannelOpened(chainID, common.HexToAddress(id), activeHermes, registryAddress)
+	if err != nil {
+		return fmt.Errorf("could not check channel status: %w", err)
+	}
+	if opened {
+		m.st.markAsMigrated(chainID, id)
+		return nil
+	}
+
```

Comment:
also maybe instead of doing this checks we could just call isMigrationRequired?

Reply:
Is migration required breaks for client who never did a migration and never called these endpoints. We need a sure way of ensuring the migration is either done or was never required even if you call this 2 years from now and all these services dont exist anymore and cannot provide you with any data.

Reply:
Then it will execute some code twice. I suggest to remain it as is.